File: app/src/main/java/com/mnj/java_solutions/gfg/ValueEqualToIndex.java
package com.mnj.java_solutions.gfg;

public class ValueEqualToIndex {

    public void valuesEqualToIndex() {
        int[] a = {15, 2, 3, 12,5};

        for (int i = 0; i < a.length; i++) {
            if (a[i] == i+1) {
                System.out.println("==>> Value and Index equal :" + a[i]);
            }
        }

    }
}

File: app/src/main/java/com/mnj/java_solutions/gfg/ValueEqualToIndex.kt
package com.mnj.java_solutions.gfg

class ValueEqualToIndex {
    fun valuesEqualToIndex() {
        val a = intArrayOf(15, 2, 3, 12, 5)
        for (i in a.indices) {
            if (a[i] == i + 1) {
                println("==>> Value and Index equal :" + a[i])
            }
        }
    }
}